theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts

```diff
@@ -44,10 +44,16 @@ export namespace TraceViewerToolbarCommands {
     };
     export const OPEN_TRACE: Command = {
```

Comment:
Maybe rename OPEN_TRACE command and id to OPEN_TRACE_FOLDER. I always seem to get confused with all the Open Trace Command references in the codebase

Reply:
Yeah it happens to me as well - i will rename them